### ISSUE TITLE
Actual fix behaviour of rule MD014 different from documentation

### ISSUE THREAD

**Issue by fpuga** [rvben/rumdl#473]
### rumdl version

0.1.31

### Installation method

Other

### Bug description

Doc of the rule [MD014](https://rumdl.dev/md014/#correct) (`commands-show-output`) says

> This rule cannot be automatically fixed

But when fixing the `$` is removed from the command.

### Steps to reproduce

1. Create  a file `foo.md` with this content:

```markdown
# Foo

```bash
$ echo "Hello, World!
```
```

2. Run the check command to verify there is a problem

```
rumdl check --no-config foo.md 
foo.md:4:1: [MD014] Command 'echo "Hello, World!' should show output (add example output or remove $ prompt) [*]

Issues: Found 1 issues in 1 file (21ms)
Run `rumdl fmt` to automatically fix 1 of the 1 issues
```

3. Run with `--fix`

```
rumdl check --no-config --fix foo.md 
foo.md:4:1: [MD014] Command 'echo "Hello, World!' should show output (add example output or remove $ prompt) [fixed]

Fixed: Fixed 1/1 issues in 1 file (27ms)
```

4. Check foo.m

```markdown
# Foo

```bash
echo "Hello, World!
```
```

### Expected behavior

The file is keep untouched.



### Actual behavior

The `check` command reports the issue as fixable, that `check --fix` modifies the file.

### Configuration

```toml

```

### Operating system

Linux

**Comment by rvben** [comment]
Thanks for the report @fpuga! The documentation was incorrect - MD014 does have auto-fix support (it removes the `$` prompt from commands without output), matching markdownlint-cli behavior.

Fixed in v0.1.33:
- Updated the MD014 documentation to accurately describe the auto-fix behavior
- If you prefer to keep the `$` prompt and add output manually, you can now use:

```toml
[global]
unfixable = ["MD014"]
```

Or via CLI: `--unfixable MD014`
